_posts/2021-11-11-java-release-notes.md

```diff
@@ -46,49 +46,21 @@ public static List<Car> findCarsOldWay(List<Car> cars){
 ```java
-public static List<Car> findCarsWithModelOldWay(List<Car> cars){
-    List<Car> selectedCars = new ArrayList<>();
-    for(Car car: cars){
-        if(car.kilometers < 50000 && car.model.equals("Mercedes")){
-            selectedCars.add(car);
-        }
-    }
-    return selectedCars;
+public static List<Car> findCarsUsingLambda(List<Car> cars){
+    return cars.stream().filter(car -> car.kilometers < 50000).collect(Collectors.toList());
 }
 ```
-We added a new condition to the `if` statement. This helps us move into our next set of examples, where we will see how to use a Lambda expression on the stream API and as the method argument.
-
-#### The World After Lambda Expressions
-We have the same problem as in the [previous example](#world-before-lambda-expressions). Our client wants to find all cars with some criteria. Let us see a little less rigid solution to this problem:
-```java
-public interface Criteria<T>{
-    boolean evaluate(T t);
-}
+This code shows how to use stream API with a lambda expression. Since we have a list of cars we need to transfer it into a stream by calling the `stream()` method. Inside the `filter()` method we are setting our condition. Every entry inside the list will be evaluated against this condition and only those that have less than 50000 kilometers will be left inside our new list. The last thing that we need to do is to wrap it up into a list.
 
-public static List<Car> findCarsUsingLambdaCriteria(List<Car> cars, Criteria<Car> criteria){
-    List<Car> selectedCars = new ArrayList<>();
-    for(Car car: cars){
-        if(criteria.evaluate(car)){
-            selectedCars.add(car);
-        }
-    }
-    return selectedCars;
-}
-
-List<Car> criteriaLambda = findCarsUsingLambdaCriteria(cars,
-    (Car car) -> car.kilometers < 500000 && car.model.equals("Mercedes"));
-```
-The first thing that we want to create is a functional interface. We can pass that functional interface as a parameter to our filtering function and call upon its method to evaluate criteria. The last line of our code shows how we use a Lambda expression to pass the implementation of the functional interface method.
-
-More about Lambda expressions can be found in the [docs](https://docs.oracle.com/javase/tutorial/java/javaOO/lambdaexpressions.html).
+More about lambda expressions can be found on [the oracle page.](https://docs.oracle.com/javase/tutorial/java/javaOO/lambdaexpressions.html)
 
 ### Method Reference
-#### Usecase
-We still own a car dealership shop, and we want to print out all cars in the shop. For that, we will use a method reference. A method reference allows us to call functions in classes using a special kind of syntax `::`. Let us see how to do it using the standard method call:
+#### Usecase Showcase
```

Comment:
Please check for consistency next time. In one place you wrote "Use Case", in another "Usecase".